api/v1alpha4/types.go

```diff
@@ -116,6 +120,28 @@ type SubnetFilter struct {
 	NotTagsAny      string `json:"notTagsAny,omitempty"`
 }
 
+type PortOpts struct {
+	NetworkID           string              `json:"networkId" required:"true"`
```

Comment:
is there any value to adding a SubnetID field? 
I would seem overkill, but there always seems to be a case where it would be nice to easily lookup the subnetID based on port.

Reply:
Also, if your goal is to support SR-IOV, then you will need to be able to set portSecurity on the ports.